core/store/migrations/migrate_test.go

```diff
@@ -1,5 +1,6 @@
 package migrations_test
 
+/*
```

Comment:
doesn't make sense to spend the effort to migrate the v1 migrations to gorm v2, perhaps I just remove this file?

Reply:
Yes please.